web-common/src/features/dashboards/time-dimension-details/util.ts

```diff
@@ -12,22 +12,6 @@ export function transposeArray(arr, rowCount, columnCount) {
   return columnarBody;
 }
 
-/**
- * This formatter is less strict and
- * returns undefined if the input is undefined
- * */
-export function safeFormatter(formatter) {
-  return (value) => {
-    if (value === undefined) {
-      return undefined;
-    } else if (value === null) {
-      return null;
```

Comment:
We still need the ability to return `null` as output when the input is `null`. The core formatter returns "null" (a string) for `null` input.

With string passed down, the data table shows the string as is 
![image](https://github.com/rilldata/rill/assets/4402679/74232289-f8cb-45a2-a8e4-6c4435b22182)

Reply:
ah ok, thanks @djbarnwal, I'll update that!

Reply:
fixed!